requirements/constraints.in

```diff
@@ -26,4 +26,6 @@ Pillow<10.0.0
 # AttributeError: 'ResourcePath' object has no attribute 'collection'
 Office365-REST-Python-Client<2.4.3
 # NOTE(christine) Pinned to set the `unstructured-inference` version
-unstructured-inference==0.5.13
\ No newline at end of file
+unstructured-inference==0.5.15
+# NOTE(crag) pinning to avoid error in argilla library
```

Comment:
FYI @cragwolfe I think you pinned this, right? I put your name for context.

Reply:
ah, that was for pydantic below i think. it's also been quite awhile. i.e., not related to unstructured-inference. is it causing issues?